samples/go/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.16.4
+RUN mkdir /app
+COPY bin/leadIntegration /app
```

Comment:
you should not copy the binary from outside of the image, you have to generate it in a previous layer, eg https://github.com/olxbr/casio-api/blob/master/Dockerfile#L14

Reply:
Updated Dockerfile to build app